lib/actions/create.js

```diff
@@ -13,21 +11,13 @@ const Boom = require('boom');
  *
  */
 
-module.exports = function createRecord(route, options){
+module.exports = function createRecord(route, options) {
 
-    return (request, reply) => {
+    return async (request, h) => {
 
         const actionUtil = require('../actionUtil')(request, options);
```

Comment:
I wonder why this is required in here 🤔

Reply:
Not sure what you mean.  How would you expect to see it required?

Reply:
I would expect something like `const ActionUtil = require('../actionUtil')` at the top of the file.